spec/integration/ClientSpec.php

```diff
@@ -29,7 +29,6 @@ function let(){
 
         $this->beConstructedWith([
             'baseUrl'       => getenv('NOTIFY_API_URL'),
-            'serviceId'     => getenv('SERVICE_ID'),
```

Comment:
Could be left in to ensure backwards compatibility - we haven't yet migrated over... :(